lib/builderator/interface/packer.rb

```diff
@@ -28,6 +28,12 @@ def initialize(*_)
               b[:tags] = Config.profile.current.tags
             end
 
+            # If we specify encrypted boot, packer won't allow ami_users.
+            # See: https://github.com/mitchellh/packer/pull/4023
```

Comment:
See also https://github.com/mitchellh/packer/pull/4023#issuecomment-256296753:

> Note that when using this feature we should disable the check that prevents AMI Sharing when `AMIEncryptBootVolume` is `true` because in this case the AMI can be shared (given that the KMS key is also appropriately shared with the target account through IAM.
>
>https://github.com/mitchellh/packer/pull/4023/files#diff-5dba49c9dc76437bfbfbd2e93dade275R59

If I'm not mistaken implementing the suggestion there will allow us to enable *both*.